gallery/src/components/NxTable/NxTableFilterExample.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow
+} from '@sonatype/react-shared-components';
+import { toLower } from 'ramda';
+
+const initialState = [
+  {name: 'Anna', country: 'USA'},
+  {name: 'Lean', country: 'France'},
+  {name: 'Louis', country: 'France'},
+  {name: 'Zach', country: 'Colombia'},
+  {name: 'Jimmy', country: 'Germany'}
+];
+
+const NxTableFilterExample = () => {
+
+  const [rows, setRows] = useState(initialState);
+  const [filter, setFilter] = useState('');
+
+  function getCountries(list: { name: string; country: string }[]) {
+    return list.map((state: any) => {
+      return state['country'];
+    });
+  }
+
+  const countries = getCountries(initialState);
+  const [optionsDropdown, setOptionsDropdown] = useState<Set<string>>(new Set(countries));
+
+  const onFilterChange = (filter: string) => {
+    setFilter(filter);
+    if (filter !== '') {
+      setRows(rows.filter((row: any) => toLower(row.name).includes(toLower(filter))));
+    }
+    else {
+      setRows(initialState);
+    }
+  };
+
+  const onDropdownLinkChange = (filterDropdown: string) => {
+    if (filterDropdown !== '') {
+      if ('All' !== filterDropdown) {
```

Comment:
Once I've chosen a country... What should I do? left all the countries in the list ? I left the one chosen because only this values will be in the rows
But I added this 'All' option in order to set the rows to the initial state
![image](https://user-images.githubusercontent.com/57679720/95598802-96e23280-0a50-11eb-8e64-238760fc99d1.png).